lib/tapioca/compilers/symbol_table/symbol_generator.rb

```diff
@@ -364,9 +364,13 @@ def method_missing(symbol, *args)
             end
 
             define_singleton_method(:include) do |mod|
-              before = singleton_class.ancestors
-              super(mod).tap do
-                mixins_from_modules[mod] = singleton_class.ancestors - before
+              begin
+                before = singleton_class.ancestors
+                super(mod).tap do
+                  mixins_from_modules[mod] = singleton_class.ancestors - before
+                end
+              rescue Exception # rubocop:disable Lint/RescueException
+                # this is a best effort, bail if we can't perform this
```

Comment:
Might be nice to `warn` here, otherwise this "silent failure" could make future debugging difficult. @paracycle

Reply:
Agreed in principle, but I would like to address the few places where we unconditionally and silently rescue errors similar to this into a more uniform framework. The output is already pretty noisy and I would not like to make it more noisy by surfacing these errors as is.

I had created the #238 issue to handle that in the DSL generation part of the codebase, but will most certainly extend the same thing to other parts of the codebase too. It will probably be a file-based logging operation with an indication in the UI that there might have been some errors encountered during the operation, so best to inspect the file.